src/main/java/gov/nci/ppe/controller/UserController.java

```diff
@@ -436,6 +436,14 @@ private String verifyToken(String authString) {
 					throws JsonProcessingException {
 		
 		List<User> newUsersList = userService.insertDataFetchedFromOpen(openResponseDTO);
+		newUsersList.stream().map(Participant.class::cast).forEach(patient ->{
+			try {
+				raiseInsertParticipantAuditEvent(patient.getPatientId());
+			} catch (JsonProcessingException jsonProsException) {
+				logger.log(Level.WARNING, jsonProsException.getMessage());
+			}
+		});
```

Comment:
Put this in the UserService. This would be part of each individual record creation. The list may contain both new and updated patient records, and the audit event will have to be different.